### ISSUE TITLE
A11y_VisualStudioCodeClient_ Sidebar Bar Options_ Accessing The Git side bar options_Name Role Value: Read our docs is a link but Voice over is announcing as button

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/vscode#191074]
[[Check out Accessibility Insights!](https://accessibilityinsights.io/)]- Identify accessibility bugs before check-in and make bug fixing faster and easier.
### GitHubTags:
#A11yWCAG2.1; #A11yTCS; #A11ySev3; #Visual Studio Code Client; #BM-VisualStudioCodeClient-MAC-Aug2023; #Voiceover; #MAC; #Name Role Value; #WCAG4.1.2.
### Environment and OS details:
Visual Studio Code insider for MAC version 1.82.0
Mac Version: 13.5
Screen reader: Voice Over
### Repro Steps:
1.	Open Visual Studio Code Insiders editor
2.	Using "Ctrl+Shift+G" key navigate to the search side view bar option.
3.	Using "Tab" key navigate to Git side bar options.
4.	Turn on Voice over and Observe that Read our docs is a link but Voice over is announcing as button.
### Actual Result:
Read our docs is a link but Voice over is announcing as button.
### Expected Result:
Read our docs is a link and Voice over should be announcing it as Link.
### User Impact:
Screen reader users may get confused if role of link is incorrectly defined as a button.
### Attachment:

https://github.com/microsoft/vscode/assets/98728920/3af350cc-2af9-4c9c-87fd-057d1e56f2a7

**Comment by meganrogge** [comment]
This isn't really a bug because "read our docs" makes it clear the user will be taken somewhere if they action the button, but would be good to fix if possible

**Comment by isidorn** [comment]
I have just now looked deeper into the issue. And this is VoiceOver behaviour. VoiceOver the screen reader can decide how it wants to read certain elements. So it is an upstream issue that we can not fix. 
We are not putting a button role on the element, it is a plain link.

Thus closing this as upstream against VoiceOver.

**Comment by rperez030** [comment]
This issue describes a behavior related to Voiceover on macOS, but I'm observing the same in all screen readers. All of them in Windows or mac identify the control labeled as "read our docs" as a button. However, I don't believe this should be considered as a compliance blocking bug in this context since it doesn't have any real user impact. a link or a button in this context would behave exactly the same, and it is clear by the label what the control does when activated. I recommend downgrading this to a sev3, which the product team can later close as having no user impact if they decide to do so.

**Comment by isidorn** [comment]
@rperez030 thanks for chiming in 🙏  I agree.
Let me reopen this issue then. And we can change the severity to Sev3.

**Comment by keerthiduvvuri** [comment]
As per SME guidance decreasing the bug severity to A11ySev3.
